### DISCUSSION TITLE
[Ribbon] Hide Quick Access Toolbar

### DISCUSSION THREAD

**Discussion by herledant** [Krypton-Suite/Standard-Toolkit#1332] [Q&A]
Hello every body,

I try to hide the QAT but i don't find any option for.
The QATLocation to hidden propertie doesn't hide the toolbar but just button.

![image](https://github.com/Krypton-Suite/Standard-Toolkit/assets/9166449/feb14b16-015c-4d52-ab3e-af8b6d67e8ae)

Did i miss something?

Regards

**Comment by Smurf-IV** [comment]
Hello,
Different themes have different settings for the display of QAT...
Get the latest alpha's of both toolkit and demos and see how the Ribbon examples do their thing..

**Reply by herledant** [reply]
Hello smurf-IV,

I think this IS the allowformChrome that IS responsable of this but it seems that IS not work with the kryptonform extended.

**Reply by Smurf-IV** [reply]
You have mentioned ` kryptonform extended.` Does this mean that you are not using the `KryptonForm`, but the one from the _Extended toolkit_ ?

**Comment by PWagner1** [comment]
Hi @herledant 

Did this resolve your issue?
